.github/workflows/build.yml

```diff
@@ -28,6 +28,10 @@ jobs:
     runs-on: ubuntu-latest
     env:
       BUILDKIT_INLINE_CACHE: 1
+      CACHE_IMAGE: "ghcr.io/ibm/text-gen-router:build-cache"
+      CACHE_REGISTRY: "ghcr.io"
+      QUAY_ORG: "quay.io/wxpe/text-gen-router"
```

Comment:
the variable name `QUAY_ORG` indicates its value should be `"wxpe"`
maybe `SERVER_IMAGE` would fit better?

Reply:
🌶️  yeah I think I meant to put `QUAY_REPOSITORY` here since it's the repo to push to on quay